server/py/services/api/tests/unit/db/test_artifacts.py

```diff
@@ -43,6 +43,7 @@
 
 class TestArtifacts(TestDatabaseBase):
     def test_list_artifact_name_filter(self):
+        project = "some-project"
```

Comment:
Since this file is using a test class, you can define the project name as a class variable and access it in all tests using `self.project` instead of defining it again in every test.